be/src/olap/tablet_manager.h

```diff
@@ -149,55 +131,58 @@ class TabletManager {
     // Return OLAP_SUCCESS, if run ok
     //        OLAP_ERR_TABLE_INSERT_DUPLICATION_ERROR, if find duplication
     //        OLAP_ERR_NOT_INITED, if not inited
-    OLAPStatus _add_tablet_unlock(TTabletId tablet_id, SchemaHash schema_hash,
-                         const TabletSharedPtr& tablet, bool update_meta, bool force);
-    
-    OLAPStatus _add_tablet_to_map(TTabletId tablet_id, SchemaHash schema_hash,
-                                 const TabletSharedPtr& tablet, bool update_meta, 
-                                 bool keep_files, bool drop_old);
-    
-    void _build_tablet_stat();
-    bool _check_tablet_id_exist_unlock(TTabletId tablet_id);
-    OLAPStatus _create_inital_rowset(TabletSharedPtr tablet, const TCreateTabletReq& request);
-    
+    OLAPStatus _add_tablet_unlocked(TTabletId tablet_id, SchemaHash schema_hash,
```

Comment:
I think you could add code guide for the tablet manager class, to let other developers know how to name a method.

Reply:
Done.